charmhelpers/contrib/openstack/context.py

```diff
@@ -1775,3 +1775,19 @@ def __call__(self):
                     ctxt['memcache_server_formatted'],
                     ctxt['memcache_port'])
         return ctxt
+
+
+class ExtraPolicyContext(OSContextGenerator):
```

Comment:
I may be missing something here, but the This Context's purpose purely seems to be for its side-effect?  I'm guessing that a template also uses 'policyd' at the top level to indicate the directory to whatever is reading the policies.

In this case, therefore, I think we need an explanation of how this policy is to be used with an example in the class' docstring, please.